nucypher/blockchain/eth/clients.py

```diff
@@ -264,6 +272,19 @@ def peers(self):
     def unlock_account(self, address, password):
         return self.w3.geth.personal.unlockAccount(address, password)
 
+    def sign_transaction(self, transaction: dict) -> bytes:
+
+        # Do not include a 'to' field for contract creation.
+        if transaction['to'] == b'':
+            transaction = dissoc(transaction, 'to')
+
+        # Sign
+        result = self.w3.eth.signTransaction(transaction=transaction)
+
+        # Return RLP bytes
+        rlp_encoded_transaction = result.raw
```

Comment:
Any reason, in particular, we don't just return `result.raw` directly?

Reply:
It's a stylistic preference to give the return variable a name, this also provides a place for a breakpoint. :-)

Reply:
👌